src/settings/settings.js

```diff
@@ -0,0 +1,22 @@
+import "./settings.scss";
+import bodyStyles from "../styles.scss";
+import {
+  setDataInStorage,
+  getDataFromStorage,
+} from "../utils/localStorage/localStorage.js";
+
+const SAVE_BUTTON = document.querySelector(".section__button--save");
+const INPUT = document.querySelector(".section__input--playerName");
+
+const onbuttonClick = () => {
+  setDataInStorage("playerName", INPUT.value);
+};
+
+function getPlayerNameIfExists() {
+  if (localStorage.length > 0) {
```

Comment:
This test is a weak one. To be honest it would be better to make it like that:
`const playerName = getDataFromStorage("playerName")
if (playerName) {
  INPUT.value = playerName;
}`

You can't be sure there are no other things saved in `localStorage`.

Reply:
yes, i know that, but only something like this went to my mind. thanks for help, I'll change that tomorrow